console/module/user/pom.xml

```diff
@@ -42,6 +42,10 @@
             <groupId>org.eclipse.kapua</groupId>
             <artifactId>kapua-user-api</artifactId>
         </dependency>
+         <dependency>
```

Comment:
Please remove this dependency

Reply:
Hi @Coduz , as @lorthirk proposed me to make a new private method in `GwtUserServiceImpl`  , I had to add dependency that is needed for getting Devices from DeviceQuery. So If I remove this dependeny, this solution will not work.

Reply:
Ah right, I didn't see that! :D Sorry